Backpack/Calendar/Classes/BPKCalendarStickyHeader.m

```diff
@@ -62,6 +102,40 @@ - (void)setMonth:(NSDate *)month {
                                                                    content:self.titleLabel.text
                                                                  textColor:appearance.headerTitleColor];
     self.titleLabel.attributedText = monthText;
+
+    BPKCalendar *calendar = [self bpkCalendar];
+    if (!calendar) {
+        return;
+    }
+
+    self.selectMonthButton.hidden = !calendar.allowsWholeMonthSelection;
+    BPKSimpleDate *simpleMonth = [BPKSimpleDate simpleDatesFromDates:@[month] forCalendar:calendar.gregorian].firstObject;
+    self.selectMonthButton.enabled = [calendar isWholeMonthButtonEnabledForMonth:simpleMonth];
+    [self.selectMonthButton setTitle:calendar.wholeMonthTitle];
+    self.selectMonthButton.accessibilityLabel =
+        [NSString stringWithFormat:@"%@ %@", calendar.wholeMonthTitle, [[BPKCalendarStickyHeader formatter] stringFromDate:month]];
```

Comment:
If we can read the month label first then "Select whole month" would be sufficient if it is highlighted right after. 
No need to mash the month and button together

Reply:
Just wanted to make it easier for people with impaired vision.